code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -27,10 +28,24 @@
 					to_chat(M, "<span class='warning'>The nullrod's power interferes with your own!</span>")
 					M.mind.vampire.nullified = max(5, M.mind.vampire.nullified + 2)
 
+/obj/item/nullrod/pickup(mob/living/user)
+	. = ..()
+	if(sanctify_force && (!user.mind || !user.mind.isholy))
+		user.adjustBruteLoss(force)
+		user.adjustFireLoss(sanctify_force)
+		user.Weaken(5)
+		user.unEquip(src, 1)
+		to_chat(user, "<span class='warning'>[src] slips out of your grip as you pick it up, bouncing upwards and smacking you in the face!</span>")
```

Comment:
Glorious

Reply:
Yeah. I'm gonna add a sound effect (punch) to that so it sounds like it, too.

Reply:
Bahaha, excellent! I'm excited to see this one in action.